src/bindgen/library.rs

```diff
@@ -55,10 +55,16 @@ impl Library {
 
     pub fn generate(mut self) -> Result<Bindings, Error> {
         self.remove_excluded();
-        self.functions.sort_by(|x, y| x.path.cmp(&y.path));
         self.transfer_annotations();
         self.simplify_standard_types();
 
+        match self.config.function.sort_by {
+            SortStradegy::ByName => {
+                self.functions.sort_by(|x, y| x.path.cmp(&y.path));
```

Comment:
I think this can just be `sort_by_key(|x| &x.path)`, while you're at it.

Reply:
Thanks. Learnt something new here.